src/components/molecules/TablesControlBar/TablesControlBar.scss

```diff
@@ -0,0 +1,8 @@
+@import "scss/constants.scss";
+
+$control-bar-height: 40px;
+
+.TablesControlBar {
+  height: $control-bar-height;
+  margin-top: 13px;
```

Comment:
Can you use our global constants here for this value?

Reply:
Can't find any relevant var in global constants for the value of `13px`. Anytime `13px` is used in the app it's just set in .scss file

Reply:
Is there a margin constant that is close to it? Do we know why it's specifically 13px?

Reply:
Yeah, I'm using `$spacing--md` (12px) instead of. Well, 13px was in Figma design, and I hope it's fine to use 12px